doc: add minutes for meeting 17 Sep 2020 (#19)
nodejs/next-10: meetings/2020-09-17.md

### nodejs/next-10

* Next 10 years for collaborators [#16](https://github.com/nodejs/next-10/issues/16)
  * added by Mary, one of the things we should consider as next step  

* Prioritizing Technical Values [#5](https://github.com/nodejs/next-10/issues/5)
  * Values/directions document for core [#2](https://github.com/nodejs/next-10/issues/2)
    * PR open don’t need more discussion https://github.com/nodejs/node/pull/35145

* Discussion on what our next major focus area should be.  
  * what we should do on the collaborator front [#16]
  * steps toward technical priorities

* We spent most of the time discussion the steps toward technical priorities.
  * Discussion around how we get feedback from the right people get the right people involved
  * Discussion on constituencies and what is important to them as next step for defining
    Technical priorities. We should be able to map the priorities back to the constituencies.
    * Noticed gap in that we are missing enterprises as a constituency
    * We brainstormed the following addigion
        * Organizations with investments in Node.js (ex: Enterprises, Government bodies, startups, non-profits) 
    * Darcy will PR in.

>   * Discussed that ext focus area should be what does each constituencies need

Author: ```suggestion
* Discussed that next focus area should be what does each constituencies need
 
```

# Node.js  Next 10 Years team Meeting 2020-09-17

## Links

> * **Recording**:  https://youtu.be/xeXjCK4fCIA

Reviewer: nit
```suggestion
* **Recording**: https://youtu.be/xeXjCK4fCIA
```

* **GitHub Issue**: https://github.com/nodejs/next-10/issues/18

## Present
* Michael Dawson (@mhdawson)
* Wes Todd (@wesleytodd)
* Darcy Clarke (@darcyclarke)
* Ruy Adorno (@ruyadorno)
* Jean Burellier (@sheplu)
* Joe Sepi (@joesepi)

## Agenda

## Announcements
 
*Extracted from **next10-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/next-10

* Next 10 years for collaborators [#16](https://github.com/nodejs/next-10/issues/16)
  * added by Mary, one of the things we should consider as next step  

* Prioritizing Technical Values [#5](https://github.com/nodejs/next-10/issues/5)
  * Values/directions document for core [#2](https://github.com/nodejs/next-10/issues/2)
    * PR open don’t need more discussion https://github.com/nodejs/node/pull/35145

* Discussion on what our next major focus area should be.  

>   * what we should do on the collaborator front [#16]

Reviewer: ```suggestion
  * what we should do on the collaborator front [#16](https://github.com/nodejs/next-10/issues/16)
```

  * steps toward technical priorities

* We spent most of the time discussion the steps toward technical priorities.
  * Discussion around how we get feedback from the right people get the right people involved
  * Discussion on constituencies and what is important to them as next step for defining
    Technical priorities. We should be able to map the priorities back to the constituencies.
    * Noticed gap in that we are missing enterprises as a constituency

>     * We brainstormed the following addigion

Reviewer: addition?
```suggestion
    * We brainstormed the following addition
```

>         * Organizations with investments in Node.js (ex: Enterprises, Government bodies, startups, non-profits) 

Reviewer: ```suggestion
        * Organizations with investments in Node.js (eg: Enterprises, Government bodies, startups, non-profits) 
```

    * Darcy will PR in.
* Discussed that next focus area should be what does each constituencies need

>  

Reviewer: ```suggestion
```

>     * This should actually include #16 as the Node.js maintainers are one of the constituencies
>        listed.

Reviewer: ```suggestion
    * This should actually include [#16](https://github.com/nodejs/next-10/issues/16) as the Node.js maintainers
    are one of the constituencies listed.
```

    * Next meeting to brainstorm on what each constituency needs

>     * Action -> Darcy to setup boards, for each constituency , with following headings:need, don’t

Reviewer: ```suggestion
    * Action -> Darcy to set up boards, for each constituency, with following headings: need, don’t
```
